src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -16,62 +16,157 @@ redirects:
   - /docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries
 ---
 
-import queriesnrqlAlertConditions from 'images/queries-nrql_screenshot-full_alert-conditions.png'
+import queriesPolicyNewAlertCondition from 'images/queries-nrql_screenshot-crop_create-alert-condition.png'
 
-import queriesnrqlNrqlContextDemo from 'images/queries-nrql_screenshot-full_nrql-context-demo.gif'
+import queriesMetricExplorer from 'images/queries-nrql_screenshot-crop_metrics-create-alert-condition.png'
 
 import queriesnrqlSignalLossUi from 'images/queries-nrql_screenshot-full_signal-loss-ui.png'
 
 import queriesnrqlFineTuneSignals from 'images/queries-nrql_screenshot-full_fine-tune-signals.png'
 
-You can use [NRQL queries](/docs/insights/new-relic-insights/using-new-relic-query-language/using-nrql) to create alert conditions. Once you've defined your signal, you can further define your warning and critical threshold levels. This determines when an alerts incident is created.
+You can use [NRQL queries](/docs/insights/new-relic-insights/using-new-relic-query-language/using-nrql) to create alert conditions. Once you've defined your signal, you can further define your warning and critical threshold levels. This determines when an alerts incident is created. For more information on key concepts relating to NRQL alert conditions and streaming alerts, see [Streaming alerts: key terms and concepts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/streaming-alerts-key-terms-concepts).
 
-Read on to learn more about how to do this.
+Ready to get started? If you haven't already, create your free New Relic account below to start monitoring your data today.
```

Comment:
```suggestion
```

Reply:
I would just delete this because based on some VSU squad feedback, these don't really generate sign-ups and can appear clunky.